app/controllers/prior_authority/steps/further_information_controller.rb

```diff
@@ -0,0 +1,38 @@
+module PriorAuthority
+  module Steps
+    class FurtherInformationController < BaseController
+      include MultiFileUploadable
+
+      def edit
+        @form_object = FurtherInformationForm.build(record, application: current_application)
+        @explanation = record.information_requested
```

Comment:
This might be neater if we made it available as a method on @form_object rather than its own variable. Same for `@supporting_documents`.